arkouda/dataframe.py

```diff
@@ -3694,7 +3981,7 @@ def sort_values(self, by=None, ascending=True):
             i = self.coargsort(by, ascending=ascending)
         else:
             raise TypeError("Column name(s) must be str or list/tuple of str")
-        return self[i]
+        return self._get_rows(i)
```

Comment:
I was just wondering why we switched to using `_get_rows` here when most of the other cases like this use `iloc`